### ISSUE TITLE
Forseti installer takes too long - cloud shell disconnects.

### ISSUE THREAD

**Issue by MartenvanWezel** [GoogleCloudPlatform/forseti-security#1294]
Perhaps no code changes needed but perhaps there is some advice we can give on how to install forseti? When cloud shell disconnect during install, it leaves forseti in a broken state.

Perhaps suggest to run it inside [screen](https://www.gnu.org/software/screen/manual/screen.html)?

**Comment by lukwam** [comment]
Our org is very big and so the process of doing the initial scan takes a long time.

The installer gets hung up [here](https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-rc1/setup/gcp/installer/forseti_installer.py#L272-L278):

```
+-------------------------------------------------------------------------------
|  Forseti Server VM Initialization
+-------------------------------------------------------------------------------

Waiting for Forseti server to be initialized... Command "gcloud compute ssh forseti-server-vm-2844 --zone us-central1-c --command tail -n1 /tmp/deployment.log --quiet" failed/timeout, retrying...
Waiting for Forseti server to be initialized...... Command "gcloud compute ssh forseti-server-vm-2844 --zone us-central1-c --command tail -n1 /tmp/deployment.log --quiet" failed/timeout, retrying...
Command "gcloud compute ssh forseti-server-vm-2844 --zone us-central1-c --command tail -n1 /tmp/deployment.log --quiet" failed/timeout, retrying...
Command "gcloud compute ssh forseti-server-vm-2844 --zone us-central1-c --command tail -n1 /tmp/deployment.log --quiet" failed/timeout, retrying...
Command "gcloud compute ssh forseti-server-vm-2844 --zone us-central1-c --command tail -n1 /tmp/deployment.log --quiet" failed/timeout, retrying...
Command "gcloud compute ssh forseti-server-vm-2844 --zone us-central1-c --command tail -n1 /tmp/deployment.log --quiet" failed/timeout, retrying...
Waiting for Forseti server to be initialized....
```

so it never gets to [create firewall rules](https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-rc1/setup/gcp/installer/forseti_server_installer.py#L159-L164) and it doesn't [display the instructions](https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-rc1/setup/gcp/installer/forseti_installer.py#L135-L151).

Logging into the VM shows that the server is running the it's doing the inventory:

`cli.pyc inventory create --import_as 20180323T224134`

and the main thing taking up CPU is:

`/usr/bin/python3 /usr/bin/google_metadata_script_runner --script-type startup`

**Comment by joecheuk** [comment]
Resolved in: #1302
